### DISCUSSION TITLE
mumu12成为mumu官方下载的默认版本后，adb端口是哪个？7555还是16384

### DISCUSSION THREAD

**Discussion by ACCodelover** [MaaAssistantArknights/MaaAssistantArknights#5045] [General]
如题

**Comment by ABA2396** [comment]
16384

**Reply by ACCodelover** [reply]
MAA里面选了mumu12连接配置后 显示的端口是7555 但是mumu12模拟器的诊断信息看adb确实是16384，而且两个端口adb都能连接

**Reply by ABA2396** [reply]
切换配置不会改端口，是自动检测出的7555吗？

**Reply by ACCodelover** [reply]
我把MAA删了重新解压 自动检测现在是16384了。之前检测到是7555，而且可以正常使用（未强制替换adb）,命令行下我手动adb connect两个端口是都可以连通的。

**Reply by ACCodelover** [reply]
mumu12自带的abd和MAA替换用的adb有什么区别吗？现在版本的mumu12我没强制替换但是也可以正常使用。
mumu12自带的adb版本：
Android Debug Bridge version 1.0.41
Version 34.0.1-9979309
MAA用的adb版本好像比这个要新一些 但是也是34.0.1-xxxxxxxx

**Reply by ACCodelover** [reply]
而且mumu12自带的adb大小比mumu6自带的版本大了很多 5810kb

**Reply by ABA2396** [reply]
能用就不用管，maa是直接从谷歌下的最新的adb，模拟器的adb可能会有自己的改动

**Reply by ACCodelover** [reply]
所以MAA文档写mumu12需要强制替换adb，可能不是必须的？mumu12好像是最近才成为mumu官网的默认下载版本，可能有些改动
